packages/fuchsia_remote_debug_protocol/lib/src/fuchsia_remote_connection.dart

```diff
@@ -654,10 +654,11 @@ class _SshPortForwarder implements PortForwarder {
     final String targetAddress = _ipV6 && _interface!.isNotEmpty
         ? '$_remoteAddress%$_interface'
         : _remoteAddress;
-    final List<String?> command = <String?>[
+    final String? sshConfigPath = _sshConfigPath;
```

Comment:
Why was this one necessary?

Reply:
The overall change or the local? The local is necessary because fields don't promote and it avoids a second null check.

Reply:
The local.  https://github.com/dart-lang/sdk/issues/44327#issuecomment-734704925 👍